scripts/run_end_to_end_all.sh

```diff
@@ -15,9 +15,11 @@ else
   ${CONDA} env update -f ./envs/development/conda/fedbiomed-node.yaml
 fi
 ${CONDA} env update -f ./envs/ci/conda/fedbiomed-researcher-end-to-end.yaml
-
-rmdir ./data
-ln -s ~/Data/fedbiomed ./data
+DATA_DIR=./data
+if [ -d ${DATA_DIR} ] && [ ! -L ${DATA_DIR} ]; then # test if it is the ${DATA_DIR} directory obtained from checkout
+  rm -fr ${DATA_DIR}
```

Comment:
It can be dangerous to do a `rm -rf` (especially as we neither set nor check our current directory and we use a name as common as "data") in case of a bug (or if someone runs it on another machine than CI host) ...

Possible hardenings could be:
1. either create a `scripts/ci` for `run_end_to_end*` scripts 
2. or have additional test so that it is more likely we are in the proper dir

```
if [ -d "./fedbiomed/common" ] && [ -d ${DATA_DIR} ] && [ ! -L ${DATA_DIR} ]; then # test if it is the ${DATA_DIR} directory obtained from checkout
```

3. or move the directory instead of `rm -rf` (this is fine if we clean the clone and redo a clone from fresh at each test, is it the case ?)

Reply:
hi @ErwanDemairy I pushed a proposed patch implementing 2. and 3. (no need of 1. then ...)

What do you think of it ?